modules/nodes-scheduler-node-selectors-project.adoc

```diff
@@ -5,59 +5,113 @@
 [id="nodes-scheduler-node-selectors-project_{context}"]
 = Creating project-wide node selectors  
 
-You can use node selectors on a project together with labels on nodes to constrain all pods created in a namespace to the labeled nodes.
+You can use node selectors in a project together with labels on nodes to constrain all pods created in that project to the labeled nodes.
```

Comment:
Ideally, this module should be split into a concept and a procedure module.

Reply:
@kalexand-rh I tend to agree. I have spent too much time on this issue. Not sure the benefit of spending more time.